packages/xstate-store/src/atom.ts

```diff
@@ -39,6 +40,30 @@ const [
 
 let activeSub: Subscriber | undefined = undefined;
 
+type AsyncAtomState<Data, Error = unknown> =
+  | { status: 'pending' }
+  | { status: 'fulfilled'; data: Data }
+  | { status: 'rejected'; error: Error };
```

Comment:
we use `status: 'done' | 'error'` in XState. It would be nice to have this unified. It's worth noting though that in XState `status: "pending"` would be `status: "active"`

Reply:
Yeah, I was thinking about this - I'm fine with unifying `fulfilling` -> `done` but I think it makes sense to have `status: 'active'` and `status: 'pending'` be semantically different

Reply:
[3bef5d8](https://github.com/statelyai/xstate/pull/5266/commits/3bef5d83c59c0d6b5b491ffc8292908c6cfc9c41)